pallets/gas/src/lib.rs

```diff
@@ -120,6 +120,19 @@ impl ValueNode {
             }
         }
     }
+
+    /// Returns the MessageId (as Origin) of the external node of current value tree.
+    /// If some node along the upstream path is missing, returns an error (tree is invalidated).
+    pub fn root_id<T: Config>(&self) -> Result<H256, DispatchError> {
```

Comment:
How is this different from the `root_origin()`? Or is it intended to replace the old one?